spring2/src/main/java/com/bates/spring/services/UserService.java

```diff
@@ -0,0 +1,25 @@
+package com.bates.spring.services;
+
+
+
+import com.bates.spring.dto.UserDto;
+import main.java.com.bates.spring.model.Users;
+import main.java.com.bates.spring.model.response.response.UserResponse;
+
+import java.util.List;
+
+public interface UserService {
+    List<UserDto> getUsers(int page, int limit);
+
+    UserDto createUser(UserDto userdto);
+
+    UserResponse getUser(Long id);
+
+    UserResponse getUsersByEmail(String email);
+
+    UserResponse updateUser(Users user);
+
+    void deleteUser(Long id);
+
```

Comment:
All of these should return UserDto(s). Never skip the dto step or your code won't be as flexible or expandable